Wire-iOS/Sources/Helpers/UIStackView+CustomSpacing.swift

```
@@ -0,0 +1,81 @@
+//
+// Wire
+// Copyright (C) 2017 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+extension UIStackView {
```

@nicorsm I was thinking about this implementation and I found the following:
1. Once initialized with `init(customSpacedArrangedSubviews:)`, the stack view would not be possible to use with other common stack view methods (for example, the distributed alignment would not make sense, adding another arranged subview would break the logic, etc.).
2. For any `UIStackView` that is not initialized with the `init(customSpacedArrangedSubviews:)`, calling `wr_addCustomSpacing(_:, after:)` would silently fail.

I think considering this it would be better to create the custom `UIView` subclass that would accommodate the  `UIStackView` and implement the 2 given methods (composition). In this way we can solve the 2 issues I underlined.